### DISCUSSION TITLE
Removal or suspension of Stale CI

### DISCUSSION THREAD

**Discussion by validcube** [LawnchairLauncher/lawnchair#3700] [General]
The Stale CI was intended to reduce noise and help organise the repository.

This isn't the case with Lawnchair, it encouraged people to violate GitHub's Community Guidelines by spamming / duplicating their comment on issues just to reset the timer; thus making the CI ineffective.

My suggestion would be 
* Increase the stale period to one year;
* Suspend the CI until the repository is active (right now, it's in maintenance);
* Remove the CI entirely;
* or Ignore update on comments

_Butttt... This might not be an issue once update via reactions is merged into stale action_

**Comment by SuperDragonXD** [comment]
I know it's a bit late, but yeah we have since disabled stale CI (well it's kinda obvious since you're part of the team now, but it's weird to just close this with no comment).
